calculator/main.py

```diff
@@ -58,18 +76,22 @@ def get_history(calc):
         ie:
         ('2016-05-20 12:00:00', 'add', (1, 2), 3),
     """
-    pass
+    
+    return calc['history']
 
 
 def reset_history(calc):
     """
     Resets the calculator history back to an empty list.
     """
-    pass
+    calc['history'] = []
+    return "History is Empty"
 
 
 def repeat_last_operation(calc):
     """
-    Returns the result of the last operation executed in the history.
-    """
-    pass
+    Returns the result of the last operation executed in the history."""
+    
+    if not calc['history']:
+        return None
+    return perform_operation(calc, calc['history'][-1][1], calc['history'][-1][2])
```

Comment:
You're already storing the result of the operation in your history so you can just return that result here. (ie. `return calc['history'][-1][-1]`) It doesn't really matter here as we're only performing simple operations, but it's something to keep in mind as since this is extensible it could potentially be performing complex and cpu-expensive operations that you don't necessarily want to have to compute again if you only want the result from the last time it was performed.